webgeocalc/calculation_types.py

```diff
@@ -699,3 +699,74 @@ def __init__(self, output_time_system='UTC', output_time_format='CALENDAR', **kw
         kwargs['output_time_format'] = output_time_format
 
         super().__init__(**kwargs)
+
+
+class GFCoordinateSearch(Calculation):
```

Comment:
`GFCoordinateSearch` vs. `PositionFinder` (or both 🤔)

Reply:
I kept `GFCoordinateSearch` but it would be nice to have a lookup table in the documentation to map the WebGeoCalc interface UI features with the API classes